src/ConnectedFields.js

```diff
@@ -47,21 +68,45 @@ const createConnectedFields = ({ deepEqual, getIn }) => {
       return this.refs.renderedComponent
     }
 
+    handleChange(name, event) {
+      const { dispatch, parse, normalize, _reduxForm } = this.props
+      const value = onChangeValue(event, { name, parse, normalize })
+
+      dispatch(_reduxForm.change(name, value))
+    }
+
+    handleFocus(name) {
+      const { dispatch, _reduxForm } = this.props
+      dispatch(_reduxForm.focus(name))
+    }
+
+    handleBlur(name, event) {
+      const { dispatch, parse, normalize, _reduxForm } = this.props
+      const value = onChangeValue(event, { name, parse, normalize })
+
+      // dispatch blur action
+      dispatch(_reduxForm.blur(name, value))
+
+      // call post-blur callback
+      if (_reduxForm.asyncValidate) {
+        _reduxForm.asyncValidate(name, value)
+      }
+    }
+
     render() {
       const { component, withRef, _fields, _reduxForm, ...rest } = this.props
-      const { asyncValidate, blur, change, focus, sectionPrefix } = _reduxForm
+      const { sectionPrefix } = _reduxForm
       const { custom, ...props } = Object.keys(_fields).reduce((accumulator, name) => {
         const connectedProps = _fields[ name ]
         const { custom, ...fieldProps } = createFieldProps(getIn,
           name,
           {
             ...connectedProps,
             ...rest,
-            blur,
-            change,
-            focus
-          },
-          asyncValidate
+            onBlur: this.onBlurFns[name],
+            onChange: this.onChangeFns[name],
+            onFocus: this.onFocusFns[name]
```

Comment:
As I can see some methods are already named like you suggested: `_fields, _reduxForm`. Probably it would work.

Sorry if my review sounded rude, I was just confused of such behavior.